evap/results/exporters.py

```diff
@@ -175,11 +184,43 @@ def export(self, response, course_types_list, include_not_enough_voters=False, i
                 percentage_participants = int((evaluation.num_voters / evaluation.num_participants) * 100) if evaluation.num_participants > 0 else 0
                 writec(self, "{}%".format(percentage_participants), "evaluation_rate")
 
+            if course_results_exist:
+                writen(self)
+                for evaluation, results in evaluations_with_results:
+                    if evaluation.course.evaluations.count() > 1:
+                        self.write_empty_cell_with_borders()
+                    else:
+                        self.write_empty_cell()
+
+                writen(self, _("Evaluation weight"), "bold")
+                for evaluation, results in evaluations_with_results:
+                    if evaluation.course.evaluations.count() > 1:
+                        writec(self, "{}%".format(evaluation.weight_percentage), "evaluation_weight")
+                    else:
+                        self.write_empty_cell()
+
+                writen(self, _("Course Grade"), "bold")
+                for evaluation, results in evaluations_with_results:
+                    if evaluation.course.evaluations.count() > 1:
+                        writec(self, evaluation.course.avg_grade, self.grade_to_style(evaluation.course.avg_grade))
+                    else:
+                        self.write_empty_cell()
+
+                writen(self)
+                for evaluation, results in evaluations_with_results:
+                    if evaluation.course.evaluations.count() > 1:
+                        writec(self, None, "border_top")
+                    else:
+                        self.write_empty_cell()
```

Comment:
i wonder whether we should simply not show individual evaluations but only whole courses in the export. i mean it's too much data for humans anyway and now it's even more

Reply:
what would a course be in the export? it's more or less just one grade. all the relevant information is in the evaluations, isn't it?

Reply:
right... i don't know. i had the impression that the only remaining use case of the export is to see whether any course is making problems. if people have to aggregate multiple columns of numbers in their heads to answer that question i think that's making that use case less user friendly. if you think the export serves its purpose just fine with multiple columns per course, then that's fine with me.

Reply:
it's basically scanning the sheet for orange and red fields when talking about the last semester - so i think everything is fine.